src/scss/_dialog.scss

```diff
@@ -0,0 +1,19 @@
+ux-dialog-overlay {
```

Comment:
Please comment this file that this changes are related to Bootstrap.

Reply:
Done